tests/ci/android/README.md

```diff
@@ -0,0 +1,24 @@
+# Android CI for AWS-LC
+AWS-LC wants to ensure that our tests and build work correctly on Android. Insipred from the CRT Team's [`AWSCRTAndroidTestRunner`](https://github.com/awslabs/aws-c-common/tree/main/AWSCRTAndroidTestRunner), this mini Android test harness is intended to be used within our Android CI with AWS Device Farm. The tests will include `crypto_test`, `urandom_test`, `decrepit_test` and `ssl_test`, referenced from regular test dimensions. This app can be tested with the docker image at `tests/ci/docker_images/linux-x86/ubuntu-20.04_android`.
+
+## Setup
+1. Assuming all the commands are being run from this folder: `cd tests/ci/android`
+2. `docker build -t ubuntu-20.04:android ../docker_images/linux-x86/ubuntu-20.04_android/`
+3. Run the docker image from root of aws-lc. The container needs access to aws-lc's source code to build.
+   ```
+   docker run -it -v `pwd`:`pwd` -w `pwd` ubuntu-20.04:android`
+   ```
+4. `cd AWSLCAndroidTestRunner`
+5. Run `./gradlew assembleDebug assembleAndroidTest` to build both the app and test apks with the AWS-LC non-FIPS debug build.
```

Comment:
This downloads Downloading https://services.gradle.org/distributions/gradle-5.6.4-all.zip can it be put into the docker file?

Reply:
Downloaded in the docker file now, and adjusted `tests/ci/android/AWSLCAndroidTestRunner/gradle/wrapper/gradle-wrapper.properties` to run on the downloaded version.